scripts/fiber/tests-failing.txt

```diff
@@ -1,151 +1,8 @@
-* renders a checkbox that is checked with an onChange with client render on top of good server markup
-* renders a checkbox that is checked with readOnly with client render on top of good server markup
-* renders a checkbox that is checked and no onChange/readOnly with client render on top of good server markup
-* renders a checkbox with defaultChecked with client render on top of good server markup
-* renders a checkbox checked overriding defaultChecked with client render on top of good server markup
-* renders a checkbox checked overriding defaultChecked no matter the prop order with client render on top of good server markup
-* renders a textarea with a value and an onChange with client render on top of good server markup
-* renders a textarea with a value and readOnly with client render on top of good server markup
-* renders a textarea with a value and no onChange/readOnly with client render on top of good server markup
-* renders a textarea with a defaultValue with client render on top of good server markup
-* renders a textarea value overriding defaultValue with client render on top of good server markup
-* renders a textarea value overriding defaultValue no matter the prop order with client render on top of good server markup
-* renders a select with a value and an onChange with client render on top of good server markup
-* renders a select with a value and readOnly with client render on top of good server markup
-* renders a select with a multiple values and an onChange with client render on top of good server markup
-* renders a select with a multiple values and readOnly with client render on top of good server markup
-* renders a select with a value and no onChange/readOnly with client render on top of good server markup
-* renders a select with a defaultValue with client render on top of good server markup
-* renders a select value overriding defaultValue with client render on top of good server markup
-* renders a select value overriding defaultValue no matter the prop order with client render on top of good server markup
-* renders a controlled text input with client render on top of good server markup
-* renders a controlled textarea with client render on top of good server markup
-* renders a controlled checkbox with client render on top of good server markup
-* renders a controlled select with client render on top of good server markup
-* should not blow away user-entered text on successful reconnect to an uncontrolled input
-* should not blow away user-entered text on successful reconnect to a controlled input
 * should not blow away user-entered text on successful reconnect to an uncontrolled checkbox
 * should not blow away user-entered text on successful reconnect to a controlled checkbox
 * should not blow away user-selected value on successful reconnect to an uncontrolled select
 * should not blow away user-selected value on successful reconnect to an controlled select
```

Comment:
These tests fail because currently I'm going through the `setInitialProperties` path which tries to set all the properties. This doesn't affect text inputs because there we use `setAttribute('value', ...)` but ideally we should always set properties so that this gets properly reset to what we think it is. But we should be explicit about we think the value should be. Either the current value or the defaultValue. I'd like to follow up on that in the next separate PR.